### PR TITLE
doc:add some doc for ulimit

### PR DESCRIPTION
**Pull request by ningmingxiao**
doc:add some doc for ulimit. I find docker doc doesn't describe  how to use these fields.
https://docs.docker.com/reference/cli/docker/container/run/#ulimit

### CODE REVIEW

File: docs/command-reference.md [hunk 1]
```diff
@@ -378,6 +378,26 @@ Ulimit flags:
...
+| memlock    | max locked-in-memory address space (KB) | same as above|
+| msgqueue   | max memory used by POSIX message queues (bytes)| same as above|
+| nice       | nice priority | same as above |
+| nproc      | max number of processes | same as above|
+| rss        | max resident set size (KB)| same as above|
+| rtprio     | max realtime priority| same as above|
+| rttime     | realtime timeout | same as above|
+| sigpending | max number of pending signals| same as above|
+| stack      | max stack size (KB) | same as above|
+| rnofile    | max number of open file descriptors| A 64-bit integer (int64), with no units. It cannot be negative; negative values will be forcibly converted to a large number, and an "Operation not permitted" error will occur during setting|
```
Comment (yankay) [inline docs/command-reference.md:399]: HI @ningmingxiao 
Do you mean `nofile`

Comment (ningmingxiao) [reply docs/command-reference.md:399]: yes ,docker doc doesn't describe how to use ulimit in detail.  https://docs.docker.com/reference/cli/docker/container/run/#ulimit

Comment (yankay) [reply docs/command-reference.md:399]: HI @ningmingxiao 
The 'rnofile' may need to change to `nofile`

Comment (ningmingxiao) [reply docs/command-reference.md]: done, thank you.

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**